### ISSUE TITLE
Not rendering full schematic 

### ISSUE THREAD

**Issue by laurenceg1099** [Lunatrius/Schematica#534]
![Minecraft 1 12 2 10_02_2021 12_09_10](https://user-images.githubusercontent.com/73744180/107515222-a3af9800-6ba2-11eb-87fb-aaf2a24d69d0.png)
![Minecraft 1 12 2 10_02_2021 12_10_57](https://user-images.githubusercontent.com/73744180/107515228-a4e0c500-6ba2-11eb-98a5-3d8cf4334736.png)
as you can see in these pictures it has the purple box, showing it should be much larger however it stops abruptly, this is a large farm design and it may have problems because of the size, I have tried loading it on both a server and singleplayer world and it happens on both, I have tried getting the schematic again and it seems not to work. I even changed the Mods render distance and still does not work, it is a perfectly straight edge which leads me to believe that it was not saved properly. This happens on multiple different schematics of a similar size.
 any help would be appreciated
cheers, 
Laurence

**Comment by Furiouspotato** [comment]
yeah similarly, I've been having the same problem - its really annoying, and I haven't been able to find a fix yet :/
one thing tho, for that missing part, you might be able to do it in chunks, as I'm trying to make a giant farm like that too, so thats what im gonna do
